### PR TITLE
DNS kickstart options for RHEL 8

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Sort-of-but-not-really-port of #4571 to RHEL-8.

This is a nearly complete rewrite, many thanks to @rvykydal's advice and guidance with ifcfg.

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
There's no coverage for most of this in rhel 8, I'm afraid. There is a kickstart test, though...

File: pyanaconda/modules/network/ifcfg.py [hunk 1]
```diff
@@ -372,6 +372,16 @@ def get_kickstart_network_data(ifcfg, nm_client, network_data_class, root_path="
     if iface:
         kwargs["device"] = iface
 
+    # DNS search etc.
+    if ifcfg.get("DOMAIN"):
+        kwargs["ipv4_dns_search"] = ifcfg.get("DOMAIN")
+    if ifcfg.get("IPV6_DOMAIN"):
+        kwargs["ipv6_dns_search"] = ifcfg.get("IPV6_DOMAIN")
```
[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply pyanaconda/modules/network/ifcfg.py:379]: I think so, will re-test :O

[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply pyanaconda/modules/network/ifcfg.py]: Yes, this was wrong, fixed and retested.

**Comment by jkonecny12** [comment]
After https://github.com/rhinstaller/anaconda/pull/4754 is merged please add revert commit to return back the testing of network kickstart command.

**Review by rvykydal — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**